frontend/src/component/filter/Filter.css

```diff
@@ -0,0 +1,23 @@
+.filter {
```

Comment:
use rems instead of px while refactoring

Reply:
I will change it to rem, thanks @humzahasan .